.gitignore

```diff
@@ -6,4 +6,5 @@ Gemfile.lock
 .#*
 .env
 log/
-tmp/
\ No newline at end of file
+tmp/
+.idea
```

Comment:
IDE files should be in a global .gitignore:

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore